resources/docker-cleanup.sh

```diff
@@ -5,6 +5,9 @@ function jssh(){
 }
 
 for HOST in $(echo $DOCKER_HOSTS | sed "s/,/ /g"); do
-	jssh ${HOST} "docker images | grep '<none>' | awk '{print \$3}' | xargs -r docker rmi"
-    jssh ${HOST} "docker ps -aq | xargs -r docker rm -f"
+	# jssh ${HOST} "docker images | grep '<none>' | awk '{print \$3}' | xargs -r docker rmi"
+  # more agressive
+  jssh ${HOST} "docker images -q | xargs -r docker rmi -f"
```

Comment:
We'll have to re-fetch fresh images each day. Is this side-effect intentional?

Reply:
@kahowell basically yes.  I think we can fine-tune the frequency this happens by modifying the trigger?